src/bt.py

```diff
@@ -0,0 +1,106 @@
+import struct
+import binascii
+import bluetooth
+from micropython import const
+import ble_advertising as blea
+
+
+_IRQ_SCAN_RESULT = const(5)
+_IRQ_SCAN_DONE = const(6)
+
+_GAP_SCAN_INTERVAL_US = const(128_000)
+_GAP_SCAN_WINDOW_US = const(11_250)
+_GAP_ADV_INTERVAL_US = const(100_000)
+
+class LampBluetooth:
+    MAGIC_NUM = const(42069)
+    COLORS_FIELD_FORMAT = "<H3B3B"
+    @classmethod
+    def _hex_to_rgb(cls, color: str):
```

Comment:
i should probably follow this _ prefix convention elsewhere since it appears to be the pythono way to infer "internal use" !